### PR TITLE
Resources: Show 'No Search Results' Text

### LINKED ISSUE #624: Resources: Add 'No Search Results' Text when empty object returned
Show a message when no resources of a particular category are returned 

Issue to be solved similar to the Events and Projects case

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #624

### CODE REVIEW

**Comment by akshatnitd** [comment]
@alanmoo Please review :)

File: app/react/pages/resources/resources.jsx [hunk 1]
```diff
@@ -114,7 +117,8 @@ export default React.createClass({
           <div className="row">
             {resourceCards}
           </div>
-          <div className="text-xs-center">
+          <div className="lead text-xs-center mb-2">
+            {this.state.empty && <p>Sorry, no results were found in this category</p>}
             <button hidden={this.state.allPagesLoaded} className="btn btn-outline-info mb-3" onClick={this.onMoreClick}>See More</button>
```
Comment (arush0311) [inline app/react/pages/resources/resources.jsx]: seems good to me. but do we want to propogate lead class style to underlying button?

Comment (akshatnitd) [inline app/react/pages/resources/resources.jsx]: Can be removed and a separate div can be created for the error text. Should I make the changes and send amend the commit ?

Comment (arush0311) [inline app/react/pages/resources/resources.jsx]: I think so. Perhaps the class can be added to p tag

Comment (akshatnitd) [inline app/react/pages/resources/resources.jsx]: @arush0311 Made the changes :+1:

File: app/react/pages/resources/resources.jsx [hunk 2]
```diff
@@ -114,7 +117,10 @@ export default React.createClass({
           <div className="row">
             {resourceCards}
           </div>
-          <div className="text-xs-center">
+          <div className="text-xs-center mb-2">
+            <div className="lead">
+              {this.state.empty && <p>Sorry, no results were found in this category</p>}
+            </div >
```
Comment (arush0311) [inline app/react/pages/resources/resources.jsx]: why didn't you use mb-2 and lead on p tag?

**Pull request merged**